src/basic/errno-to-name.awk

```diff
@@ -3,7 +3,7 @@
 BEGIN{
```

Comment:
`s/errno-to-awk/errno-to-name/` in the commit message :)

Reply:
Oops. But I don't think it's worth pushing and retriggering the CI for this.